### ISSUE TITLE
FEAT: MAKE SOCIAL ANCHORS ACCESSIBLE ON FOOTER 💡 

### ISSUE THREAD

**Issue by 0xabdulkhalid** [Eduhub-Community/Eduhub-Community.github.io#516]
### Is your feature request related to a problem? Please describe.

- This feature is related to the aspect of accessibility problem

### Describe the solution you'd like

- The solution is to add important attributes like `aria-label` for social anchors (links like Github) which are placed on Social media section.

- it is important to note that we want to add the `title` attribute also. This is because the `title` attribute is intended to provide additional information to screen reader users.

- The `aria-label` attribute is a way more important for providing accessibility information to screen reader users. The `aria-label` attribute is an Accessible Rich Internet Applications (ARIA) attribute that can be used to provide a label for an element. This label is then read aloud by screen readers, providing additional information about the element to the user.

- In general, the `aria-label` & `title` attributes are preferred for providing accessibility information to screen reader users.

- You can consider to assign this issue to me to make the site accessible for screen readers

### Describe alternatives you've considered

- There's no alternative's for accessibility improvements

### Additional context

- I'm a GSSoC`23 Contributor

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by 0xabdulkhalid** [comment]
Hey there @AshAman999 👋🏻,

Please make sure to assign this issue to me as soon as possible to make the work done quickly.

Thank you!
